README.md

```diff
@@ -14,7 +14,7 @@ The text reporter writes `probable root cause` text to the output, if that packa
 
 ## NPM version support
 
-This package supports NPM 7 and 8 from `2.0.0`. If you want to use it with NPM 6, then install the latest [`1.x`](https://github.com/team-supercharge/audit-ci-wrapper/tree/v1.x) version.
+This package requires at least NPM 7 since `2.0.0`. If you want to use it with NPM 6, then install the latest [`1.x`](https://github.com/team-supercharge/audit-ci-wrapper/tree/v1.x) version.
```

Comment:
The 2.0.0 version is not true. Update it to `>3.0.0`.

Reply:
We are requiring npm versions in 2.0.0:

https://github.com/team-supercharge/audit-ci-wrapper/compare/v1.0.2...v2.0.0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

Reply:
We require npm 7 and npm 8 there. From now on (in the next release = version bigger than 3.0.0) we require npm 7-8-9.

Reply:
![image](https://github.com/team-supercharge/audit-ci-wrapper/assets/5883922/e6b2299c-2439-41f1-952f-1c227b9942aa)